### ISSUE TITLE
how to initialize memory on app startup

### ISSUE THREAD

**Issue by EderSantana** [feathersjs/feathers-memory#8]
How do I initialize the memory on app startup? I tried the following without success:

```javascript
(function() {
  var app, bodyParser, feathers, memory;

  feathers = require('feathers');

  memory = require('feathers-memory')();

  bodyParser = require('body-parser');

  app = feathers();

  app.configure(feathers.rest())
    .use(bodyParser.json())
    .use('/values', memory).
    configure(feathers.errors())
    .listen(3000);

  console.log('App listening on 127.0.0.1:3000');

  // Here is what I tried
  memory.create({
    name: 'initial-value',
    type: "test",
    x: []
  });

}).call(this)
```

I also tried passing an object right after the `require`. Do you already have a solution for that? Thanks in advance.

**Comment by EderSantana** [comment]
Solution in #5 

I should have passed all the parameters with json formart:

```javascript
// create(data, params, callback), although I don't know yet what params are used for...
memory.create({
    "name": 'initial-value',
    "type": "test",
    "x": []
  }, {}, function(error, data){
    console.log('created data: ', data)
});
```

**Comment by daffl** [comment]
Yep. This is how it looks like:

```js
var feathers = require('feathers');
var memory = require('feathers-memory')();
var bodyParser = require('body-parser');
var app = feathers();

app.configure(feathers.rest())
  .use(bodyParser.json())
  .use('/values', memory)
  .listen(3000);

console.log('App listening on 127.0.0.1:3000');

app.service('values').create({
    name: 'initial-value',
    type: "test",
    x: []
}, function(error, data) {
  console.log('Created', data);
});
```

You actually shouldn't have to pass the callback though, I'll have to look into why this is happening.

**Comment by EderSantana** [comment]
I case that might help, here is the error I got:

```
/Users/eder/js/agnez/node_modules/feathers-memory/lib/memory.js:79
    cb(null, data);
    ^
TypeError: undefined is not a function
    at Object.Proto.extend.create (/Users/eder/js/agnez/node_modules/feathers-memory/lib/memory.js:79:5)
    at Object.self.(anonymous function) [as create] (/Users/eder/js/agnez/node_modules/feathers-memory/node_modules/uberproto/lib/proto.js:61:21)
    at Object.<anonymous> (/Users/eder/js/agnez/app.js:17:10)
    at Object.<anonymous> (/Users/eder/js/agnez/app.js:50:4)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
    at startup (node.js:129:16)
```

**Comment by daffl** [comment]
Related: https://github.com/feathersjs/feathers-commons/issues/4

**Comment by daffl** [comment]
This should be fixed now via https://github.com/feathersjs/feathers/pull/151 and will be available in the upcoming 1.2 release.
